src/routes/NavigatorRouter.res

```diff
@@ -109,6 +113,13 @@ let make = () => {
       })
       ->ignore
     }
+    if nativeProp.clientSecret->String.split("_")->Array.get(0)->Option.getOr("") == "pm" {
+      setAllApiData({
+        ...allApiData,
+        savedPaymentMethods: None,
```

Comment:
Can we add `paymentType: "normal"` here? this is required to actually render the payment-sheet & clear the loading state.

```
additionalPMLData: {
          ...allApiData.additionalPMLData,
          paymentType: Some("normal"),
        },
```